lib/lwm2m-server.js

```diff
@@ -103,6 +104,29 @@ function validateTypes(config, callback) {
     callback(error);
 }
 
+/**
+ * If Lifetime Resource exists, then the registration SHOULD be removed by the Server if a new registration or update
+ * is not received within this lifetime.
+ */
+function checkLifetime() {
+    var CHECK_INTERVAL = 1000;
```

Comment:
Do you really think this should be a part of user configuration and not be done automatically on the background? I would prefer to config OMA Objects where is the default server lifetime defined and so on.